pkg/endpoints/category/associate.go

```diff
@@ -0,0 +1,48 @@
+package category
+
+import (
+	"fmt"
+	"github.com/discmonkey/retext/pkg/db"
+	"log"
+	"net/http"
+)
+
+func AssociateEndpoint(store db.Store) func(w http.ResponseWriter, r *http.Request) {
+	t := func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodPost {
+			http.Redirect(w, r, "/", http.StatusSeeOther)
+			return
+		}
+
+		categoryID := r.FormValue("category")
+		documentID := r.FormValue("key")
+		text := r.FormValue("text")
```

Comment:
Will Formvalue still work without a traditional form actually being submitted? Just wondering.

Reply:
Dunno. The docs read like it can handle a lot. But googling says not json... will the data be passed as json?

Reply:
Guess you're going to have some fun trying it :)